hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/execution/TestSimpleExecutionInSpark.java

```diff
@@ -88,10 +80,10 @@ public Integer finish() {
             return count;
           }
         };
-    SimpleHoodieExecutor<HoodieRecord, Tuple2<HoodieRecord, Option<IndexedRecord>>, Integer> exec = null;
+    SimpleExecutor<HoodieRecord, HoodieRecord, Integer> exec = null;
 
     try {
-      exec = new SimpleHoodieExecutor(hoodieRecords.iterator(), consumer, getCloningTransformer(HoodieTestDataGenerator.AVRO_SCHEMA));
+      exec = new SimpleExecutor<>(hoodieRecords.iterator(), consumer, Function.identity());
```

Comment:
Similar here for `getTransformer`.  The production code passes the transfer in by calling `getTransformer` so let's follow the same in the test code.